README.md

```diff
@@ -1 +1,177 @@
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML file]**
+- **[CSS file]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [N/A](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+In order to run this project you need:
+1. Knowledge HTML and CSS
+
+2. VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+
+3. Node package. [Link to download node](https://nodejs.org/en/download/)
+
+### Setup
+
+Clone this repository to your desired folder:
+use this command to clone my repository "git clone"
+add this link(https://github.com/SolomonTheCoder/Microverse-Portfolio)
+
+### Install
+
+Install this project with:
+
+- npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+- npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+- npm test
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/SolomonTheCoder)
+- Twitter: [@twitterhandle](https://twitter.com/SolomonAfutu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/solomon-afutu-164b95255/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[New HTML file]**
+- [ ] **[New CSS file]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project... I encourage to support me by following me on github and thanks for your support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me this opportunity to become a software developer. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
As previously requested, this should link to your personal license for this project instead of toa place to choose a license from. This problem stems from the fact that you don't have the license file itself to link to. So I suggest adding a file named MIT.md and then add the contents from [here](https://github.com/mutugi-c/Awesome-books/blob/main/MIT.md). Please add your name here and after that, come over to the read me and replace `[MIT](https://choosealicense.com/licenses/mit/)` with `[MIT](./MIT.md)` in order to make the documentation complete.